README.md

```diff
@@ -1 +1,198 @@
+
+E) In the navbar I can see a button to show the list of all the reservations created by the logged-in user, with the name of the property, reservation Id, and a button to delete it.
+
+## Built With:
+This project was built using these technologies:
+* React
+* Redux & Redux Toolkit
+* HTML
+* Sass
+* JavaScript
+
+## Usage:
+
+To have this app on your pc, you need to:
+
+  - Clone with SSH:
+  ```
+git@github.com:Home-Sharing-Team/final-capstone-Project-Back-End.git
+  ```
+  - Clone with HTTPS
+  ```
+https://github.com/Home-Sharing-Team/final-capstone-Project-Back-End.git
+  ```
+
+* In the project directory, you can run:
+
+  - `$ npm install` - installs all the dependencies required by the project
+
+  - Then, run the following command on the root project's directory to run the application:
+  ```
+  $ npm start
+  ```
+
+## Testing:
+
+To test the application, run the following command on the root project's directory:
+ ```
+  $ npm test
+  ```
+
+## Kanban Board:
+
+Link to the project Kanban board: <a href="https://github.com/orgs/Home-Sharing-Team/projects/2">Click here</a>.
+
+Link to the screenshot of the initial state of the Kanban board: <a href="https://user-images.githubusercontent.com/64566209/199840132-dde39cfe-be8a-4172-9bf5-f5d0485ce13f.png">Click here</a>.
+
+
+## Authors:
+
+👤 **Ulises Largaespada**
+
+- GitHub: [@ulisesedu7](https://github.com/ulisesedu7)
+
+- Twitter: [@LongswordMusic](https://twitter.com/LongswordMusic)
+
+- LinkedIn: [Ulises Largaespada](https://www.linkedin.com/in/ulises-largaespada-45570b1a4/)
+
+
+👤 **Gabriel Santo**
+
+- GitHub: [@gvgesanto2](https://github.com/gvgesanto2)
+
+- Twitter: [@GabrielSanto997](https://twitter.com/GabrielSanto997)
+
+- LinkedIn: [Gabriel Santo](https://linkedin.com/in/gabriel-santo-5882a71b2/)
+
+
+👤 **David Verde Alvarez**
+
+- This is My **GitHub**: [![@David-Verde](https://img.shields.io/github/followers/omarramoun?label=David&style=social)](https://github.com/David-Verde)
+
+- This is My **Twitter**: [![@Unyielding1](https://img.shields.io/twitter/follow/omarramoun?label=David16&style=social)](https://twitter.com/UnyieldingOne)
+
+
+👤 Dario Alessio
+
+- This is My **GitHub**: [![@DarioAlessioR](https://img.shields.io/github/followers/omarramoun?label=Dario&style=social)](https://github.com/DarioAlessioR)
+
+- This is My **Slack**: https://microverse-students.slack.com/team/U039GCFRK9B
+
+- This is My **LinkedIn**: https://www.linkedin.com/in/dario-alessio-3a3b7911b
+
+
+## Show your support:
+
+Give a :star: if you like this project!
+
+
+## Acknowledgements:
+
+- Based on the [design](https://creativecommons.org/licenses/by-nc/4.0/) by The Creative Commons
+- Hat tip to anyone whose code was used
+- Microverse
+- Our supportive families 😊
+
+## License:
+This project is [MIT](https://github.com/David-Verde/final-capstone-Project-Back-End/blob/develop/LICENSE) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪